rest/importuserxattrtest/revcache_test.go

```diff
@@ -73,7 +73,6 @@ func TestUserXattrRevCache(t *testing.T) {
 
 	resp := rt.SendAdminRequest("PUT", "/{{.keyspace}}/"+docKey, `{}`)
 	rest.RequireStatus(t, resp, http.StatusCreated)
-	require.NoError(t, rt.WaitForPendingChanges())
```

Comment:
For this (and a few other places where WaitForPendingChanges is being removed) - we're sure this isn't needed to ensure the document hits the channel cache before making the xattr update below?  It's possible this was originally here from an (unnecessary) copy/paste, but I can't tell for sure just looking at it.

Reply:
Since this is testing the revcache explicitly, it probably should be here even though the test will pass without it. I'll take a pass and put this back to the way it was.